crawl-ref/source/dat/des/arrival/large.des

```diff
@@ -1115,3 +1115,57 @@ MAP
       xxxxxxGxxxxxx
         xxxxxxxxx
 ENDMAP
+
+##############################################################################
+# 39x22
+NAME:   beargit_arrival_diablo_church
+TAGS:   arrival no_trap_gen no_item_gen no_monster_gen
+ORIENT: float
+MONS:   rat / nothing w:50
+FTILE:  `t = floor_grass, 1'G_> = floor_limestone
```

Comment:
You should correspondingly have a line:
`COLOUR:  ` = green, 1'G = white` so that console players get a good experience.